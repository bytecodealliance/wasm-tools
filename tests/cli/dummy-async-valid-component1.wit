// RUN: component embed % --dummy-names legacy --async-callback | \
//        component new | \
//        component wit

package root:root;

world root {
  type xe = u32;

  export name: func() -> xe;
}
