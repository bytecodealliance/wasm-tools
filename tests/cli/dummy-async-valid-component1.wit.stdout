package root:component;

world root {
  type xe = u32;

  export name: func() -> xe;
}
